schema.sql

```diff
@@ -21,4 +21,30 @@ CREATE TABLE owners (
 CREATE TABLE species (
   id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
   name VARCHAR(250) NOT NULL
-);
\ No newline at end of file
+);
+
+CREATE TABLE vets (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(250) NOT NULL,
+  age INT NOT NULL,
+  date_of_graduation DATE NOT NULL
+);
+
+CREATE TABLE specializations (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	vets_id INT NOT NULL,
+	species_id INT NOT NULL,
+	FOREIGN KEY (vets_id) REFERENCES vets (id) ON DELETE RESTRICT ON UPDATE CASCADE,
+	FOREIGN KEY (species_id) REFERENCES species (id) ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+
+CREATE TABLE visits (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  vets_id INT NOT NULL,
+	animals_id INT NOT NULL,
+  date DATE NOT NULL,
+	FOREIGN KEY (vets_id) REFERENCES vets (id) ON DELETE RESTRICT ON UPDATE CASCADE,
+	FOREIGN KEY (animals_id) REFERENCES animals (id) ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
```

Comment:
- [ ]  Kindly fix this issue I am getting when trying to create 2 tables `visits` and the `specialization` . Check here :point_down: 

![image](https://user-images.githubusercontent.com/65068771/154573066-4d128314-9672-4a8a-bf78-88807c7f0a1a.png)

Reply:
Hello, @andersonlebon I think the problem with the `set` word before the foreign key. But my code doesn't have it.

![image](https://user-images.githubusercontent.com/35535130/154576444-d1ba3b76-2515-4f6b-977e-eae90e395b3d.png)